internal/storage/storage.go

```diff
@@ -89,6 +84,24 @@ func (s *Storage) UploadFile(file io.ReadSeeker, fileID string) error {
 	return nil
 }
 
+// DownloadFile downloads a file from the storage by the given id.
+func (s *Storage) DownloadFile(fileID string) (io.ReadSeeker, error) {
```

Comment:
not sure if returning a `io.ReadSeeker` is a good thing here. Ok for now, but check with bogdan

Reply:
> not sure if returning a `io.ReadSeeker` is a good thing here. Ok for now, but check with bogdan

I think this is a good solution in my case, since getting an object from S3 has a response.Body, which is io.ReadCloser, and to upload this file, the payload takes a Body, which is io.ReadSeeker.

Reply:
> not sure if returning a `io.ReadSeeker` is a good thing here. Ok for now, but check with bogdan

As an alternative, I can use [uploader](github.com/aws/aws-sdk-go/service/s3/s3manager) , which accepts io.Reader as input, but I have already implemented it differently

Reply:
it's fine